src/main/java/org/jenkinsci/test/acceptance/po/ToolInstallation.java

```diff
@@ -78,7 +78,7 @@ public static <T extends ToolInstallation> T addTool(Jenkins jenkins, Class<T> t
         final ConfigurablePageObject page = ensureConfigPage(jenkins);
 
         final String name = type.getAnnotation(ToolInstallationPageObject.class).name();
-        final Control expandButton = page.control(by.button(name + " installations..."));
+        final Control expandButton = page.control(by.button(name + " installations"));
```

Comment:
I am confused as to how is this (and other changes) compatible with the current Jenkins version given this should not match the existing button?

Reply:
The locator does it based off of contains string:
https://github.com/jenkinsci/acceptance-test-harness/blob/51712c7c3c63bbc09e7b2175cb5e38eecd638ccc/src/main/java/org/jenkinsci/test/acceptance/ByFactory.java#L170